src/vue/backend/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@idxbrokerllc/idxstrap": "^3.2.1",
+    "@idxbrokerllc/idxstrap": "^3.3.0",
```

Comment:
Version 3.3.0 is not on npm, this change will be done once we publish the new version